tests/features/support/steps.js

```diff
@@ -223,11 +225,59 @@ When(/I check the "([^"]+)" checkbox/, async (label_text) => {
   *    Very limited. Anything more is a future feature.
   * 
   * "checkbox": label that contains checkbox-like behavior.
+  * 
+  * May switch to using the below instead - almost same code, but
+  *    its text has to match exactly and it turns out clicking labels
+  *    works for more than one thing.
   */
   let checkbox = await scope.page.waitFor( `label[aria-label*="${ label_text }"]` );
   await checkbox.click();
 });
 
+When('I pick the {string} option', async (label_text) => {
+  /* Clicks the first label "containing" the "label text".
+  *    Very limited. Anything more is a future feature.
+  */
+  let choice = await scope.page.waitForSelector( `label[aria-label="${ label_text }"]` );
```

Comment:
Should this be `label[aria-label*="${ label_text }"]` (asterisk-equal instead of just equal) like in line 233?

Reply:
`*=` means that the text is contained in it. I'm transitioning to needing the exact text (which is `=`). Otherwise people can get stuck where they can't get the element they want because some other element contains a substring of the element they want. It'd be fine if it were obvious to detect, but I don't think it is.

What do you think? Is it more desirable to have more flexibility?

Reply:
I think I got confused by the comment above this line which says `Clicks the first label "containing" the "label text".` In my mind, containing == subset of, not equal to. In any case, the code is fine. No changes needed.